### PR TITLE
Add ConnectX-8 to supported NICs list

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

**Review by ykulazhenkov — APPROVED** [review]

**Review by SchSeba — COMMENTED** [review]
Hi @e0ne can you also add it to the hw supported table?

**Comment by e0ne** [comment]
> Hi @e0ne can you also add it to the hw supported table?

done

**Review by adrianchiris — APPROVED** [review]
LGTM

File: doc/supported-hardware.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ The following SR-IOV capable hardware is supported with sriov-network-operator:
 | Mellanox MT28908 Family [ConnectX-6 Dx] | 15b3 | 101d |
 | Mellanox MT28908 Family [ConnectX-6 Lx] | 15b3 | 101f |
 | Mellanox MT2910 Family [ConnectX-7 | 15b3 | 1021 |
+| Mellanox CX8 Family [ConnectX-8 | 15b3 | 1023 |
```
Comment (SchSeba) [inline doc/supported-hardware.md:25]: Can you please update the table below with the card's supported capabilities?

Comment (e0ne) [reply doc/supported-hardware.md:25]: done

Comment (wizhaoredhat) [inline doc/supported-hardware.md:25]: NIT: Typo , missing end square "]" bracket.

Comment (wizhaoredhat) [reply doc/supported-hardware.md:25]: Actually for the CX-7 as well.

Comment (e0ne) [reply doc/supported-hardware.md:25]: fixed

**Review by adrianchiris — APPROVED** [review]

**Review by SchSeba — APPROVED** [review]

**Pull request merged**